app/Models/Cases.php

```diff
@@ -4,11 +4,16 @@
 
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
+use App\Models\User;
 
-class CovidProvince extends Model
+
+class Cases extends Model
```

Comment:
The `Cases` should changed to `Case`, so Laravel will know automatically the table is `cases`.

Reply:
The name "Case" is reserved by PHP.  
I can't make model name Case.